tests/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+#
+# Tests from ghidra repo
```

Comment:
Just a general comment, but I think all the CMake files should have the license notice on them.

Reply:
I've always wondered about this, especially as it relates to infrastructure code and CMake. Sure, some of our CMake files are probably non-trivial and original, like the CMake function we distribute to compile arbitrary sleigh files or maybe the commands/setup to download the Ghidra source code, but a lot of our other files feel like they all follow the same best practices when creating a redistributable project that can be easily customized and configured.

How do you know when something is complex enough to warrant a copyright/license notice that others _must_ follow? Or do you just blanket apply and deal with the fallout if someone calls you out on it?

Reply:
Yeah good point. To be honest, I was just cargo culting from our other `lifting-bits` projects where we add copyright notices to the CMake configuration. 

Still though, doesn't this issue apply to regular code too? A lot of codebases include some kind of utilities module that contain little helpers that probably exist in other codebases in some shape or form. And as far as I can remember, those have never omitted copyright headers just because they're not complex enough to warrant it.

Personally, I think it's fine to just blindly apply it. In the cases where the code is too general, I don't think our claim would be enforceable anyway. I understand your point though, so if you want to selectively apply the notice to just the files with novel functionality in them, I'm happy with that too.

Reply:
Applied it to the files that were relatively complex